design/baremetal-operator/resume-interface.md

```diff
@@ -0,0 +1,90 @@
+<!--
+ This work is licensed under a Creative Commons Attribution 3.0
+ Unported License.
+
+ http://creativecommons.org/licenses/by/3.0/legalcode
+-->
+
+# Title
+
+Resume from error state
+
+## Status
+
+provisional
+
+## Summary
+
+This proposal aims to introduce a new declarative API
+to forcibly resume a waiting BareMetalHost resource from an error state
+when a fix is immediately available, so that it could be possible
+to reduce the waiting time required
+
+## Motivation
+
+The current BMO error retry mechanism implements an
+exponential backoff with jittering: this means that when a BMH
+goes into an error state its reconciliation loop gap increases
+as long as the error conditions persists (based on BMH `Status.ErrorCount`).
+
+In some cases it's necessary a human (or more generally an external)
+intervention to fix the issue. If the fix involves amending the
+BMH resource then an immediate reconciliation is triggered and the
+new state is evaluated again.
+Instead, if the fix does not generate a relevant event (for example,
+a firmware update on the host) captured by BMO, then it's necessary
```

Comment:
Yeah, another example could be losing network connectivity to the BMC network. This will put all of the BMHs into error backoffs, but once you restore connectivity you might want to retry immediately.

Reply:
Good case, I'll add it in the description